.teamcity/settings.kts

```diff
@@ -138,6 +140,56 @@ object BuildBaseImages : BuildType({
 	}
 })
 
+object BuildDockerImage : BuildType({
+	name = "Build docker image"
+	description = "Build docker image for Calypso"
+
+	vcs {
+		root(WpCalypso)
+		cleanCheckout = true
+	}
+
+	steps {
+		dockerCommand {
+			name = "Build docker image"
+			commandType = build {
+				source = file {
+					path = "Dockerfile"
+				}
+				namesAndTags = """
+					registry.a8c.com/calypso/app:build-%build.number%
+					registry.a8c.com/calypso/app:commit-${WpCalypso.paramRefs.buildVcsNumber}
+				""".trimIndent()
+				commandArgs = "--pull --build-arg use_cache=true"
+			}
+			param("dockerImage.platform", "linux")
+		}
+		dockerCommand {
+			commandType = push {
```

Comment:
Does this need a `name`?

Reply:
Good catch!
I don't know, I exported this directly from TeamCity UI. Maybe names are optional and I forgot to add one?

Reply:
Yep, they are optional:

![image](https://user-images.githubusercontent.com/975703/97324964-bf509600-1872-11eb-91b8-80fc51741f96.png)